src/tests/tests/asan-error-parse.rkt

```diff
@@ -0,0 +1,6 @@
+#lang racket
+
+(require "../../collects/seashell/backend/asan-error-parse.rkt")
+
+(display (asan-rewrite (port->bytes)))
```

Comment:
Please write tests and integrate them into the existing testing framework we have.